pyfftw/pyfftw.pyx

```diff
@@ -1408,6 +1436,11 @@ cdef class FFTW:
         copy the returned array.
         '''
 
+        if ortho is None:
+            ortho = self._ortho
+        if normalise_idft is None:
+            normalise_idft = self._normalise_idft
+
         if ortho and normalise_idft:
             raise ValueError('Invalid options: ortho and normalise_idft cannot'
                              ' both be True.')
```

Comment:
I think this test should be more explicit. I don't much like `None` being used as `False`.